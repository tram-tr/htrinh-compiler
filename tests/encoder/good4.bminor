"Mixed: \\t\\n\\\"\\\\\\0x20\\'Good\\' Bye!"