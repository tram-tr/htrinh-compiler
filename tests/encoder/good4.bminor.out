decoded: Mixed: \t\n\"\\\0x20\'Good\' Bye!
encoded: "Mixed: \\t\\n\\\"\\\\\\0x20\\'Good\\' Bye!"
encoding successful.
